File: src/python/_1535_Find_the_Winner_of_an_Array_Game.py
class Solution:
    def getWinner(self, arr: List[int], k: int) -> int:
        curr = arr[0]
        count = 0

        for i in range(1, len(arr)):
            if arr[i] > curr:
                curr = arr[i]
                count = 0

            count += 1

            if count == k:
                break

        return curr

File: src/_1535_Find_the_Winner_of_an_Array_Game.java
class Solution {
    public int getWinner(int[] arr, int k) {
        int curr = arr[0], count = 0;

        for (int i = 1; i < arr.length; i++, count++) {
            if (count == k)
                break;

            if (arr[i] > curr) {
                curr = arr[i];
                count = 0;
            }
        }

        return curr;
    }
}
